### ISSUE TITLE
Please add thumbnail support

### ISSUE THREAD

**Issue by dust-to-dust** [jurplel/qView#492]
Thumbnail is important. Can you add thumbnail support for Windows explorer or qView

**Comment by jurplel** [comment]
Do you mean showing thumbnails in File Explorer? That isn't under my control, and it should already work.

**Comment by BloodRaven0** [comment]
Yeah thumbnails should be handled by windows explorer itself, I don't know exactly how it works, but there should be some built-in thumbnailer, because it has always worked out of the box for me. If you don't see thumbnails, look in your windows explorer settings or ask in a windows support forum.

**Comment by jurplel** [comment]
Following up a long time later to say I found out we can supply a custom "thumbnailer" to support more formats. That would be a different issue/feature request, and kind of a large scope

**Comment by BloodRaven0** [comment]
@jurplel  SageThumbs still works pretty well for extending windows' support for thumbnailing different file formats, although it is discontinued.
I'm not sure how things are on macos and linux, but isn't thumbnailing kinda playform-specific, maybe even explorer-specific (with all the different explorers on linux). That would be quite the undertaking for something people can already do with other software and it might interfere with the aforementioned software.

**Comment by jurplel** [comment]
> ...isn't thumbnailing kinda platform-specific...

> That would be quite the undertaking...

Yes, exactly
